idseq_dag/util/m8.py

```diff
@@ -445,7 +432,7 @@ def call_hit_level_v2(hits):
 @command.run_in_subprocess
 def generate_taxon_count_json_from_m8(
         m8_file, hit_level_file, e_value_type, count_type, lineage_map_path,
-        deuterostome_path, cdhit_cluster_sizes_path, output_json_file, output_json_file_compat=None):
+        deuterostome_path, taxon_whitelist_path, cdhit_cluster_sizes_path, output_json_file, output_json_file_compat=None):
```

Comment:
Can we do `taxon_whitelist_path=None` as an optional argument? (like how it was done before) Then you don't need to pass in `None` for taxon_whitelist_path in run_alignment_remotely.

Reply:
I'm now no longer passing an explicit None so kept the arguments the same, thank you for the pointer though